schedule/virt_autotest/tumbleweed-virt-guest-install.yaml

```diff
@@ -3,6 +3,7 @@ description:    >
     Maintainer: jcao@suse.com
     Virtualization guest installation tests including extended feature tests schedule
 schedule:
+    - autoyast/prepare_profile
```

Comment:
Please don't add this module in this test. These tests use autoyast profile directly instead of autoyast templates. And currently the tests install TW with static bootscripts over ipxe, "prepare_profile.pm" is not supported, as the dymanic job ID is required.

Reply:
Thanks Julie!

Reply:
Can I know if any autoyast configation files used by qe-virt team which have `hardcode` user password?

Reply:
we use encryped user password in autoyast profiles. https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/virt_autotest/host_unattended_installation_files/autoyast_opensuse_kvm_one_disk.xml